main.cpp

```diff
@@ -0,0 +1,33 @@
+// This is a program that calculates stock information
+// Created by NAOMI BROWN on 1/31/16.
+
+#include <iostream>
+#include <iomanip>
+
+// Constants
+const int SHARES_BOUGHT = 1000;
+const int SHARES_SOLD = 1000;
+const double COMMISSION_RATE = .02;
+const double PURCHASE_PRICE = 45.50;
+const double SALE_PRICE = 56.90;
+
+// Variables
+double totalPurchasePrice = SHARES_BOUGHT*PURCHASE_PRICE;
+double totalBuyCommission = totalPurchasePrice * COMMISSION_RATE;
+double totalSalePrice = SHARES_SOLD*SALE_PRICE;
+double totalSaleCommission = totalSalePrice*COMMISSION_RATE;
+double totalProfit = (totalSalePrice-totalSaleCommission)-(totalPurchasePrice+totalBuyCommission);
+
+int main() {
+    std::cout<< "Total amount paid for stock was $"
```

Comment:
Code looks nice. I personally use a little more spacing to separate the stream insertion operator (<<) for readability.